.github/workflows/system-tests-enterprise.yml

```diff
@@ -143,14 +197,14 @@ jobs:
       run: |
         echo "mlrun_hash=$( \
           cd /tmp && \
-          git clone --single-branch --branch development https://github.com/mlrun/mlrun.git mlrun-upstream 2> /dev/null && \
+          git clone --single-branch --branch ${{ steps.select-parameter.outputs.param }} https://github.com/mlrun/mlrun.git mlrun-upstream 2> /dev/null && \
           cd mlrun-upstream && \
           git rev-parse --short=8 HEAD && \
           cd .. && \
           rm -rf mlrun-upstream)" >> $GITHUB_OUTPUT
         echo "ui_hash=$( \
           cd /tmp && \
-          git clone --single-branch --branch development https://github.com/mlrun/ui.git mlrun-ui 2> /dev/null && \
+          git clone --single-branch --branch ${{ steps.select-parameter.outputs.param }} https://github.com/mlrun/ui.git mlrun-ui 2> /dev/null && \
```

Comment:
This breaks kinda backwards compatibility as now I wouldn't be able to test on a test branch and will have to create the same branch in UI as well. I would leave it to development. same in `mlrun_hash` above